src/org/usfirst/frc/team4533/robot/Robot.java

```diff
@@ -55,9 +59,26 @@ public void robotInit() {
     	prefs = Preferences.getInstance();
     	seed = 5;
     	maxSpeed = 100;
+    	
+    	// Start up our Arduino data feed
+    	Arduino.initialize();
     }
     	
 	
+    /**
+     * This function updates our smart dashboard values. It should probably be called periodically 
+     */
+    public void updateSmartDashboard() {
+        SmartDashboard.putBoolean("Gear", DriveSystem.hasGear());
+        SmartDashboard.putBoolean("Climber On?", ClimbSystem.isOn());
+        SmartDashboard.putNumber("Front Distance", DriveSystem.ultraSonic());
+        String messageOfTheDay = "don't do school stay in drugs";
```

Comment:
Can we please remove this MotD? I get the desire to be playful, but it serves no purpose at this point in time.